jupyter/tensorflow/ubi8-python-3.8/Pipfile

```diff
@@ -8,7 +8,7 @@ name = "pypi"
 [packages]
 # TensorFlow and useful extensions
 
-tensorflow-gpu = "~=2.11.0"
+tensorflow = "~=2.11.0"
```

Comment:
Do we want to make this change in ubi8 images, given that this includes package change ?